extensions/azdata/src/common/httpClient.ts

```diff
@@ -35,6 +37,20 @@ export namespace HttpClient {
 						outputChannel.appendLine(response.statusMessage);
 						return reject(response.statusMessage);
 					}
+					const filename = path.basename(response.request.path);
+					const targetPath = path.join(targetFolder, filename);
```

Comment:
Woudl path.resolve be a better option instead of path.join?

Reply:
I don't expect to have to handle parsing paths here so leaving as is